inference-engine/src/legacy_api/src/ie_cnn_layer_builder_ngraph.cpp

```diff
@@ -382,6 +382,9 @@ CNNLayer::Ptr NodeConverter<ngraph::op::Convert>::createLayer(const std::shared_
     case Precision::FP32:
         precision_str = "FP32";
         break;
+    case Precision::FP64:
```

Comment:
It can be done separately.

I think we can replace this switch/case to one line: `precision_str = p.name();`